login.php

```diff
@@ -1,41 +1,97 @@
+		// Not not match both Admin table and User table then can't login
+		} else if($myusername !== '' and $mypassword !== '') {
+			$error = "Your username or password is incorrect";
+		}
+	}
+ob_end_flush();
+?>
+
 <!DOCTYPE html>
 <html lang = "en">
 	<head>
-		<title>Layout1Login</title>
+		<title>Login</title>
 		<meta chrset="utf-8"/>
-		<meta name="Keywords" content = "html5, layout, CSS Grid System"/>
+		<meta name="Keywords" content = "html5"/>
 		<meta name = "Author" content = "Tony Nguyen"/>
-		<meta name = "Description" content = "CSS Grid System Layout Tutorial"/>
+		<meta name = "Description" content = "Music database"/>
 		<meta name = "viewport" content = "width=device-width. initial-scale=1"/>
-		
-		<link rel ="stylesheet" href = "style5.css"> 
+		<!-- Connect to css page -->
+		<link rel ="stylesheet" href = "login.css"> 
 		
 	</head>
 <body>
+	<!--Set for grid system for the page -->
 	<div class ="grid-container">
-		<div class="img_1">
-			<img src = "images/img.png">
-		</div>
+		<!-- The div for nav and in the div it contain the logo and a function to called the nav -->
 		<div class="navigation">
+			<!-- Set a div that contain logo for the music database -->
+			<div class="logo">
+				<a href = "#"><img src = "images/logo8.png" height = "100"></a>
+			</div>
+			<!--nav for User page -->
 			<nav>
 				<ul>
-					<li><a href = "index.html">Home</a></li>
-					<li><a href = "index.html">Music Play</a></li>
-					<li><a href = "index.html">Contact</a></li>
-					<li><a href = "01_login_real.php">Account</a></li>
+					<li><a href = "homepage.php">Home</a></li>
+					<li><a href = "musicplayer.php">Music Play</a>
+						<ul>
+							<li><a href= "Song_sort_Title.php">Title</a></li>
+							<li><a href= "Song_sort_Genre.php">Genre</a></li>
+						</ul>
+					</li>
+					<li><a href = "login.php">LoginOut</a></li>
+					<li><a href = "signup.php">SignUp</a></li>
 				</ul>
 			</nav>
 		</div>
-		<div class="content"><!-- Holds the main page content -->
-			<div class="section3">
-				<p>Welcome to Music MP3</p>
-				<h3><form method = "post" id = "01_login">
-				<input id="us" type = "text" name = "username" placeholder="User name"/><br />
-				<input id="pa" type = 'password' name = 'password' placeholder="Password"/><br />
-				<input  id="login" type = "submit" value = "login"/><br />
-				</form></h3>
-			</div>
+		<div class="content"><!-- Holds the login form -->
+			<p>Welcome to Music MP3</p>
+			<h3><form method = "post">
+			<input id="us" type = "text" name = "username" placeholder="User name"/><br />
+			<input id="pa" type = 'password' name = 'password' placeholder="Password"/><br />
+			<input  id="login" type = "submit" value = "login"/><br />
+			<!--This is for User that forgot their password  -->
+			<a href = "Update_user.php" id='update'>Forgot your password?</a>
+			<p class = "error"><?php echo $error; ?></p>
+			</form></h3>
+		</div>
+		<!--Div for the image -->
+		<div class="img_2">
```

Comment:
03/08/2021: This is the img I use as t banner in the right side of the login page. So the page look balance and doesn't look to empty.